tests/transports-governance/test_governance_edge_cases_comprehensive.py

```
@@ -0,0 +1,1433 @@
+                    assert updated_budget["current_usage"] == large_usage
+
+            elif response.status_code == 400:
+                # System rejected large value - acceptable
+                pass
+
+    def test_connection_exhaustion_recovery(self, governance_client, test_data_factory):
+        """Test recovery from connection exhaustion"""
+        # Create many rapid connections
+        connection_test_operations = []
+
+        for i in range(50):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)